SortingAlgorithms/Helpers/SortedChecker.cs

```diff
@@ -8,11 +8,11 @@ namespace SortingAlgorithms.Helpers
 {
     public static class SortedChecker
```

Comment:
Please, avoid static classes. 
For me it better to call ArrayValidator.

Reply:
done